rebasehelper/output_tools/text_output_tool.py

```diff
@@ -0,0 +1,154 @@
+import six
+import os
+
+from rebasehelper.output_tool import BaseOutputTool
+from rebasehelper.exceptions import RebaseHelperError
+from rebasehelper.logger import LoggerHelper, logger, logger_report
+from rebasehelper.results_store import results_store
+
+
+class TextOutputTool(BaseOutputTool):
+
+    """ Text output tool. """
+
+    NAME = "text"
+    DEFAULT = True
+
+    @classmethod
+    def match(cls, cmd=None):
+        """
+        Checks if the given string matches the output tool
+
+        :param cmd: output tool name
+        :return: True if the name matches
+        """
+        if cmd == cls.NAME:
+            return True
+        else:
+            return False
+
+    @classmethod
+    def get_name(cls):
+        """
+        Get name of the output_tool
+
+        :return: output tool name
+        """
+        return cls.NAME
+
+    @classmethod
+    def print_success_message(cls):
+        """Print result message"""
+        results = cls.results_store.get_result_message()
+        if 'success' in results:
+            logger_report.info(results['success'])
+        else:
+            logger_report.info(results['fail'])
+
+    @classmethod
+    def print_changes_patch(cls):
+        """Print info about the location of changes.patch"""
+        patch = cls.results_store.get_changes_patch()
+        if patch is not None:
+            logger_report.info('\nPatch containing changes between old and new version patches:')
```

Comment:
This seems a bit weird, how about `Patch with differences between old and new version source files`?